src/olympia/addons/serializers.py

```diff
@@ -322,6 +323,11 @@ def to_representation(self, instance):
             repr['files'] = [repr.pop('file')]
         return repr
 
+    def get_reviewed(self, instance):
+        return serializers.DateTimeField().to_representation(
+            instance.file.approval_date or instance.human_review_date
+        )
```

Comment:
Do we want to potentially expose the human review date ? Could that reveal information that wasn't available before ? (either publicly or through API token from a developer)

Reply:
I don't think that would be necessary at this point. What matters more to developers is the approval/signing date, which I think is public already?

Reply:
It should return essentially the same value as `reviewed` was - if it was auto approved it'd be date that happened; otherwise it'd be when it was manually rejected.

I'm not sure we need to expose either date but it was already exposed as `reviewed` and this avoids an API change.